### ISSUE TITLE
Update documentation for code generation

### ISSUE THREAD

**Issue by rkrishnasanka** [stephenafamo/bob#182]
It would be great if there was a full example for:

1. Code Generation Config with all the options and explicitly listed so that we don't need to piece it together
2. Full example of using the generated code (CRUD). The documentation is a little ambiguous right now in terms of how I should establish the database connections, etc.

**Comment by stephenafamo** [comment]
I would appreciate a PR on this, or other documentation improvements in general.

**Comment by rkrishnasanka** [comment]
@stephenafamo do you have any examples I can use for the time being ?

**Comment by jacobmolby** [comment]
@rkrishnasanka Simply call `bob.New(db)`

**Comment by rkrishnasanka** [comment]
Yeah I figured it out shortly after commenting. I missed the link for bob executors. I'll do a PR once I'm done implementing stuff.

**Comment by rkrishnasanka** [comment]
Is there a discord for this ? I feel like its better to get some direct feedback. The query parameters are confusing. For example:

```
	queuesView := psql.NewView[*models.Queue]("core", "queues")
	// Check if the queue already exists
	row, err := queuesView.Query(
		context.Background(),
		bob.NewDB(db),
		models.SelectWhere.Queues.QueueName.EQ(queuename),
	).One()
```

In this scenario, I get:

```
"pq: missing FROM-clause entry for table \"queues\"" bob golang
```
as the error

**Comment by jacobmolby** [comment]
If you are using the code generation, the view/table is already created for you.

Simply use:
```golang
queue, err := models.Queues.Query(
		context.Background(),
		bob.NewDB(db),
		models.SelectWhere.Queues.QueueName.EQ(queuename),
	).One()

//queue is of type models.Queues now
```

**Comment by rkrishnasanka** [comment]
@jacobmolby So turns out that there are a couple more errors happening here that made this complicated #187 . so thats why I've just been dealing with bugs.

**Comment by pcriv** [comment]
@rkrishnasanka I couldn't find any documentationion of how to use the CLI to generate the code, could you maybe share how you did it?

**Comment by rkrishnasanka** [comment]
Here's the command I use 

```sh
go run github.com/stephenafamo/bob/gen/bobgen-psql@latest -c ./config/bobgen.yaml
```

Here's the config 
```

psql:
  dsn: "postgres://postgres:@localhost:5432/postgres?sslmode=disable"

  schemas: ["application","core"]
  shared_schema: "public"
```

**Comment by singhsays** [comment]
My PR  #469 should address some of this. I will also update the configuration options documentation in a follow up. Feel free to assign this to me @stephenafamo
